### ISSUE TITLE
[Bug] No padding around text in a Toast or Snackbar

### ISSUE THREAD

**Issue by davidortinau** [xamarin/XamarinCommunityToolkit#704]
### Description

@VladislavAntonyuk how can I set some padding around my text? I see layout options in the platform code, but I don't see how I should influence the margins, padding, etc.

![image](https://user-images.githubusercontent.com/41873/103255784-48a64680-4950-11eb-935c-c0a4b396504e.png)

### Basic Information

- Version with issue: 1.0.0-pre6
  - iOS: 14.3

### Reproduction Link

```cs
void Handle_Navigating(object sender, Xamarin.Forms.ShellNavigatingEventArgs e)
		{			
				var querystring = e.Target?.Location.OriginalString;
				if (querystring.IndexOf("?msg") > -1)
				{
					var substring = querystring.Substring(querystring.IndexOf("?"));
					var nvc = HttpUtility.ParseQueryString(substring);
					var options = new SnackBarOptions()
					{ 
						BackgroundColor = Color.FromHex("#CC0000"),
						MessageOptions = new MessageOptions
						{
							Message = nvc["msg"],
							Foreground = Color.White,
							Font =  Font.SystemFontOfSize(16)
						}
					};
					App.Current.MainPage.DisplayToastAsync(options);
				}
		}
```

**Comment by VladislavAntonyuk** [comment]
@davidortinau I will check if it is allowed on Android, because all platforms except Android use custom drawing, but Android uses native toast and snackbar.

**Comment by davidortinau** [comment]
If my Toast or Snackbar on iOS was displayed consistently with Android, I would be ok with that too. As you can see from my screenshot the iOS is using no padding at all. 

The order of options from best to worst IMO are:
* all xplat customization in Xamarin.Forms code
* customization via platform specifics or setting platform defaults
* consistent look/feel across platforms with available customization in XF code
* custom renderers or effects

**Comment by VladislavAntonyuk** [comment]
Work in progress

**Comment by AndreiMisiukevich** [comment]
Fixed in #714
